doc/admin-guide/files/remap.config.en.rst

```diff
@@ -518,20 +514,95 @@ mapping rules. (It is activated before any mappings and is never
 deactivated.) The filter `local_only` will only be applied to the
 second mapping.
 
-Implict IPAllow filter
-======================
+Special Filter and ip_allow Named Filter
+----------------------------------------
+
+If :file:`ip_allow.yaml` has a "deny all" filter, it is treated as a special filter that is applied before remapping for
+optimizaion. To control this for specific remap rules, a named filter called `ip_allow` is pre-defined. This named filter is
+activated implicitly in default. To stop applying the special rule, disable the `ip_allow` filter as shown below.
+
+::
 
-To allow control of :ref:`IP Allow<ip-allow>` it is treated as an implicitly active and named
-filter. When this filter is active IP Allow checks are done before remap. To prevent this for
-specific remap rules, this filter, named "ip_allow", must be disabled. The common way of doing this
-would be ::
+   # ip_allow.yaml
+   ip_allow:
+      - apply: in
+        ip_addrs: 198.51.100.0/24
+        action: deny
+        method: ALL
 
+   # remap.config
    .deactivatefilter ip_allow
    map ...
    map ...
    .activatefilter ip_allow
 
-Note this entirely disables IP Allow checks for those remap rules.
+Note this entirely disables :file:`ip_allow.yaml` checks for those remap rules.
+
+Evaluation Order and Matching Policy
+------------------------------------
+
+ATS evaluates multiple ACL filters in the following order:
+
+1. Special "deny all" filter in :file:`ip_allow.yaml`
+2. In-line Filter in :file:`remap.config`
+3. Named Filter in :file:`remap.config`
+4. Filters in :file:`ip_allow.yaml`
+
+When an ACL filter is found, ATS stops processing ACL filters depending on the mathcing policy configured by
+:ts:cv:`proxy.config.url_remap.acl_matching_policy`.
+
+First Explicit Match Wins Policy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the default matching policy. ATS stops processing ACL filters only if an explicit match of IP address and HTTP method is
```

Comment:
But it can be an explicit match without an IP right? Like I can match on method only? And that can still match right?

Reply:
Right. If there're no `@src_ip`, it's the same as `@src_ip=ALL` that matches with any IP.

Reply:
With Brians's comment, the new description clarifies the absence of `@src_ip` matches with any IP.